runtime/service.go

```diff
@@ -457,30 +461,33 @@ func (s *service) createVM(requestCtx context.Context, request *proto.CreateVMRe
 	}()
 
 	s.logger.Info("creating new VM")
+	s.jailer, err = newJailer(requestCtx, s.logger, string(s.shimDir), s, request)
```

Comment:
How about just passing service.config.JailerConfig? It is a bit odd to see we are passing each member of `s`, along with `s` itself. The jailer doesn't have to know the entire service object.

Reply:
originally it was like that, but there may be some fields that dictate on what or how to construct a jailer. In this case, if the the jailer being returned is a `noopJailer` it needs access to the service's shim dir.

Reply:
Right now, the shim dir is passed as `ociBundlePath` anyway.

Reply:
Yes, but the `ociBundlePath` could be something different and shouldn't be relied on for the shim dir when constructing the `noopJailer`. If that path turns out to be different, it'll break things pretty good.

Reply:
Can you clarify where ociBundlePath is coming from and under what conditions it might be different from the shimdir? It's not super clear to me.